pyproject.toml

```diff
@@ -0,0 +1,118 @@
+[project]
+authors = [
+    {name = "Webkom", email = "webkom@abakus.no"},
+]
+dependencies = [
```

Comment:
Some of the deps here are not direct, but were included in the requirements files as a way to lock sub-deps. As we have a lockfile now, these should not be here anymore.

Reply:
I only find `sentry-sdk` to be a dep of `structlog-sentry` (with no required versions), `Pygments` to be a dep of `sphinx`. Do you want these to be removed? And are you aware of any more?